crates/shared/src/bad_token/token_owner_finder.rs

```diff
@@ -84,6 +88,14 @@ pub struct Arguments {
         default_value = "",
     )]
     pub whitelisted_owners: HashMap<H160, Vec<H160>>,
+
+    /// The SeaSolver url to query the token owner pairs.
+    #[clap(long, env, default_value = "https://seasolver.dev/token_holders")]
+    pub seasolver_token_owners_url: Url,
```

Comment:
I think this should be a `Vec<Url>` as it is very imaginable that we want to have it so other external solver teams also provide token owner lists for their exotic tokens.

Also, I don't think we should provide a default URL, or enable it by default.

Reply:
Changed to `Vec<Url>`. I agree this is a good suggestion only if other solvers will not require any additional argument, like api-key or something similar.